content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -504,6 +504,22 @@ stops adjusting the target (and sets the `ScalingActive` Condition on itself
 to `false`) until you reactivate it by manually adjusting the target's desired
 replica count or HPA's minimum replica count.
 
+## Recommended Deployment configuration
```

Comment:
Maybe this:
```suggestion
### Migrating Deployments and StatefulSets to horizontal autoscaling
```
would work better?

Reply:
Ooo, good call out, I had forgotten that stateful sets can now be managed via an HPA.  I wonder if they are subject to the same problem I'm attempting to document. :thinking:

Reply:
After quick testing, I don't believe we need to bring in the conversation about StatefulSets.  This object has a differing strategy when applying updates using all Kubernetes defaults.  How about something such as this instead?

```suggestion
### Migrating Deployments  to horizontal autoscaling
```

Reply:
That could work. How come you don't also need to take special care with StatefulSets though? It might be that I don't clearly understand the shortcoming as it applies to the Deployment API.

Reply:
@sftim I've completed more testing.  We _should_ include StatefulSets.  My initial testing was using a slower image that took more time for the StatefulSet to roll through which did not show the behavior that this PR is attempting to address.  When using an image that is light, I still see the behavior.  So I'm going to use your original suggestion for this bit, and then I need to place similar documentation in the StatefulSet documentation as well.